### ISSUE TITLE
Unable to load PDF files in dotnet core 2.1 projects using libvips 8.6.5 (global installation) and NetVips 1.0.4

### ISSUE THREAD

**Issue by goarsna** [kleisauke/net-vips#14]
Hi there,

today I again got a problem loading PDF files. But this time it's using NetVips in .NET Core 2.1 projects. :D

My test scenario to reproduce:
1. Create a .NET 4.6.2 console app and a .NET Core 2.1 console app.
2. In both install NetVips 1.0.4 via nuget and set `<UseGlobalLibVips>true</UseGlobalLibVips>` in the .csproj file. (Our global libvips installation is libvips 8.6.5 downloaded from the libvips project page.)
3. Load a PDF file in the static Main method
```
internal class Program
{
	private static void Main(string[] args)
	{
		NetVips.Base.VipsInit();
		var vipsImage = NetVips.Image.NewFromFile(@"pdf1.pdf");
	}
}
```
4. In .NET 4.6.2 this works without any problems. Using .NET Core 2.1 the **is not a known fileformat exception** pops up.
> NetVips.VipsException:
'unable to load from file Q:\TestFactory\testdaten\bilder+audio+pdf\bilder\bildformate\pdf1.pdf
VipsForeignLoad: "Q:\TestFactory\testdaten\bilder+audio+pdf\bilder\bildformate\pdf1.pdf" is not a known file format
>

Is there something special to consider using .NET Core? Maybe I'm just missing a setting.

Thank you very much in advance. :)

**Comment by kleisauke** [comment]
Hello, I was able to reproduce this. It looks like `<UseGlobalLibvips>true</UseGlobalLibvips>` is ignored by .NET Core 2.1. I will investigate this further. As a workaround, I can recommend to exclude the `native` NuGet directory, like this:
```diff
--- ConsoleApp.csproj
+++ ConsoleApp.csproj
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NetVips" Version="1.0.4" />
+    <PackageReference Include="NetVips" Version="1.0.4" ExcludeAssets="native" />
   </ItemGroup>
 
   <ItemGroup>
```

By the way,
You should not call `Base.VipsInit();` in your own program. It is already called by the [ModuleInitializer](https://github.com/kleisauke/net-vips/blob/66aff775abc65736a86fe2722c72e8478da4fde8/src/NetVips/ModuleInitializer.cs#L13) once the assembly is loaded. Hopefully clarified with https://github.com/kleisauke/net-vips/commit/66aff775abc65736a86fe2722c72e8478da4fde8.

**Comment by goarsna** [comment]
Thank you very much, this did the trick: Setting `ExcludeAssets="native"` worked.

And thank you for the hint with `Base.VispInit();`.

**Comment by kleisauke** [comment]
I've just released NetVips 1.0.5 which includes a fix for this. The `ExcludeAssets="native"` workaround isn't needed anymore.
